### PR TITLE
Constrain Canvas

### LINKED ISSUE #44: Limit the size of the canvas

### PR DESCRIPTION
**Pull request by dburnsii**
Closes #44

### CODE REVIEW

File: src/ts/canvas.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ export const canvasContext = canvas.getContext('2d')!;
 export const BACKGROUND_COLOR = '#121212';
 
 export function resizeCanvas() {
-  canvas.width = window.innerWidth;
-  canvas.height = window.innerHeight;
+  canvas.width = 1280;
+  canvas.height = 720;
```
Comment (quittle) [inline src/ts/canvas.ts:8]: You don't need this function anymore. Can you just set it directly on the canvas element itself or if you need js, then just set it up once in bootstrap no need to run it on every resize.

File: index.html [hunk 2]
```diff
@@ -4,5 +4,9 @@
     overflow: hidden;
   }
 </style>
-<canvas />
+<div style="width: 100%; height: 100%; background-color: #0e0e0e">
```
Comment (quittle) [inline index.html:7]: You should be able to get this for free if you apply it on the body or html element via the style tag we have if you want.

File: index.html [hunk 3]
```diff
@@ -4,5 +4,9 @@
...
+  <div style="width: 1280; height: 720; margin: auto">
```
Comment (quittle) [inline index.html:8]: Why not just apply this directly on the canvas itself?

**Review by quittle — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (dburnsii) [reply index.html]: Well I didn't see our actual HTML tag, so I figured I'd just throw in a div to get our background color for the whole page

**Pull request merged**